test/assets/input/touches.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Touch test</title>
+  </head>
+  <body>
+    <script src="mouse-helper.js"></script>
+    <button onclick="clicked();">Click target</button>
+    <script>
+      window.result = [];
```

Comment:
don't we have the same asset for mouse? Can we unify the two, simply recording all the events for the button?

Reply:
There is a similar one for keyboard, but it has a textarea.